src/main/python/analyzers/EmotionAnalyzer.py

```diff
@@ -0,0 +1,27 @@
+import pandas as pd
+from nrclex import NRCLex
+
+
+def getEmotions(text):
+    """
+    Get emotion affect frequencies for the input text
```

Comment:
Can you add the ranges here? Like is it binary, continuous between 1 and 0, etc.

Reply:
Just wondering because it may affect of we have to do normalization/standardization or not.

Reply:
based on the emotion affect values of the first 100 tweets, the function returns a continuous value between 0 and 1.

Reply:
Okay cool. Can you just add that in the comment?

Reply:
added